<launch>

	<!--Launch Robot Controller -->
	<include file="$(find robotis_controller)/launch/robotis_another.launch">
	</include>

  <arg name="db" default="false" />
  <arg name="debug" default="false" />

  <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
  <include file="$(find robotis_manipulator_h_moveit)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>

  <!-- Given the published joint states, publish tf for the robot links -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="true" output="screen" />

<!-- Launch Robot Initializer -->
<node pkg="cat_manipulator_control" type="my_init_robot" name="init_robot" output="screen"/> 

<!-- Launch Inverse Kin Service -->
<node pkg="cat_move_to_target" type="gotoxyz_service" name="gotoxyz_service" output="screen"/> 


<!-- Launch Camera Tags -->
 <include file="$(find cat_move_to_target)/launch/camera_tags.launch"/>


<!-- Launch Tags Pose Service -->
<node pkg="cat_move_to_target" type="gettagpose_service" name="gettagpose_service" output="screen"/> 

<!-- Launch Gripper -->
<node pkg="driver_gripper" type="driver_gripper_node" name="driver_gripper" output="screen"/> 

</launch>
